website/content/specification/overview.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Overview"
+description: "Discusses the information modeling concepts used by Metaschema."
+weight: 10
+---
+
+# Overview
+
+The *Metaschema Modeling Framework* provides a means to represent an [*information model*](/specification/glossary/#information-model) for a given [*information domain*](/specification/glossary/#domain), consisting of many related [*information elements*](/specification/glossary/#information-element), in a *data format* neutral form. By abstracting information modeling away from *data format* specific forms, the Metaschema Modeling Framework provides a means to consistently and sustainably maintain an *information model*, while avoiding the need to maintain each derivative *data format* individually. By consolidating *information model* maintenance into a single representation, significant time can be saved over other approaches that require each *data format* to be maintained individually.
+
+A Metaschema-based *information model* is represented in an abstract XML-based format, called a [*Metaschema module*](/specification/glossary/#metaschema-module). A *Metaschema module* represents implementations of the individual *information elements* of the *information model* using [*definitions*](/specification/glossary/#definition). These *definitions* are bound tightly to representational forms in each supported derivative *data format*, unifying representations for a given information model. Based on the tight binding provided by the Metaschema Modeling Framework, a given *information model* can be used to deterministically generate a [*data model*](/specification/glossary/#data-model) in a *data format* specific form.
+
+The Metaschema framework currently supports XML, JSON, and YAML *data formats*. Support for YAML is limited to the subset of YAML that aligns with JSON representations.
```

Comment:
Minor, non-blocking: is there an explicitly defined subset we can allude to here or is that a very detailed oriented qualification? If it is the latter and we are just summarizing, should we not leave that out and put the detail elsewhere?

```suggestion
The Metaschema framework currently supports XML, JSON, and YAML *data formats*.
```

Reply:
OK, I just didn't know, in detail or in a summary here and point to a deeper description, explain that (I am afraid to ask but I am not sure what it is).

I am fine leaving it as-is, I marked it minor and non-blocking. :-)